requirements/prod.in

```diff
@@ -7,6 +7,6 @@ PyYAML
 click
 python-dateutil
 gvgen
-pydantic
+pydantic<2.0.0
 python-baseconv
-requests
+requests<=2.29.0 # There are issues with urllib on latest version
```

Comment:
Could you be more specific about what the issues were that caused this pin?

Reply:
Thanks. Suspicion is that this error is coming from [`vcrpy`](https://github.com/kevin1024/vcrpy/blob/defad28771d99434a1fdb3b46994b21b14c36cc7/setup.py#L49-L55) rather than `requests`.